scripts/zones/Lower_Jeuno/npcs/Treasure_Coffer.lua

```diff
@@ -34,6 +34,118 @@ local optionToKI =
     [20] = xi.ki.TONBERRY_KEY,
 }
 
+local optionToGear =
+{
+    --ACP
+    [1] = {addon = 1, itemid = 11313}, -- nuevo_coselete
+    [2] = {addon = 1, itemid = 11314}, -- mirke_wardecors
+    [3] = {addon = 1, itemid = 11315}, -- royal_redingote
+    --AMK
+    [4] = {addon = 2, itemid = 11487}, -- champions_galea
+    [5] = {addon = 2, itemid = 11488}, -- anwig_salade
+    [6] = {addon = 2, itemid = 11489}, -- selenian_cap
+    --ASA
+    [7] = {addon = 3, itemid = 16369}, -- blitzer_poleyn
+    [8] = {addon = 3, itemid = 16370}, -- desultor_tassets
+    [9] = {addon = 3, itemid = 16371}, -- tatsumaki_sitagoromo
```

Comment:
These itemid's could be added to `items.lua` and the comments could then be removed